docs/support/index.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: Support
+description: How to find answers to questions or problems you encounter while using IBM Quantum Platform or Qiskit Runtime
+---
+# Support
+
+## Get help with IBM Quantum Platform and Qiskit
+
+**Premium Plan, On-Prem Plan, Direct Access, Flex Plan, Pay-As-You-Go Plan, special programs, and Open Plan (excludes Trial accounts):**
+
+All support inquiries must be submitted by [creating a case in the IBM Cloud® Support Center](https://cloud.ibm.com/unifiedsupport/cases/form) and selecting 'Qiskit Runtime' in the Topic field. Ensure you have selected the appropriate IBM Cloud account (your institutional cloud account or any non-trial account) in the account switcher.
+
+Please reach out for any technical matter, including platform issues, job errors, Qiskit guidance, or broader quantum computing questions. Our team includes technical experts ready to assist across this full range of topics.
+
+**Community support:**
+
+Have a Trial account or looking for other support options?  [Join the Qiskit Slack community](https://qisk.it/join-slack) to get help from our large community of users and experts. In Slack you will find topical channels such as #qiskit-runtime, #ibm-quantum-platform, #qiskit-transpiler, #qiskit-ecosystem, and many more.
+
+You can also use [Stack Exchange](https://quantumcomputing.stackexchange.com/?tags=qiskit) for support from the wider community about quantum computing and programming-related questions.
+
+## Having trouble creating an account or logging in?
+
+If you get an error when you create an IBM Cloud&reg; account, first check whether you have access to the [Support Center](https://cloud.ibm.com/unifiedsupport/cases/form) on IBM Cloud.
+
+- If you have access, click "create case", then select *Account* as the Topic and *Account verification* as the Subtopic.
+- If you do not have access, open a case on the [Create an Account, Login or Billing Request](https://watson.service-now.com/x_ibmwc_open_case_app.do#!/create) page.
+
+If you are getting an error while trying to create an account and your country is on the [Personal use availability](https://cloud.ibm.com/docs/account?topic=account-account-getting-started#signup-personalaccts) list, explore the following options:
+
+1. You can use [Documentation](/docs) and [Learning](/learning) resources without an account
+2. Create a [SkillsBuild account](https://www.ibm.com/academic/) if you are a university student
+3. [Create a Trial account](https://quantum.cloud.ibm.com/registration) for 30 days of free access (Note that you will likely lose access at the end of the trial)
+
+<Admonition type="caution" title="Note">
+If you are the **owner** of a Pay-As-You-Go or Subscription account, you need to create an IBM Cloud account and provide billing information for verification.
+
+If you are **invited to join** an account, simply accept the email invitation and follow the provided steps to gain access without providing billing information.
```

Comment:
Do you want to link to https://quantum.cloud.ibm.com/docs/en/guides/cloud-setup-invited ?

Reply:
Ooo we have that now, don't we!